tests/modules/bodfeedback/test_bodfeedback.py

```diff
@@ -0,0 +1,129 @@
+import flask
+from donut.modules.bodfeedback import helpers
+from donut.testing.fixtures import client
+from datetime import datetime
+
+
+def test_get_link(client):
+    assert helpers.get_link(
+        1) == flask.url_for('home') + 'bodfeedback/view/sample_uuid'
+    assert helpers.get_link(500) is None
+    return
+
+
+def test_get_id(client):
+    assert helpers.get_id('sample_uuid') == 1
+    assert helpers.get_id('nonsense') == False
+    return
+
+
+def test_get_messages(client):
+    messages = helpers.get_messages(1)
+    assert messages[0]['time'] == datetime(2018, 1, 1, 0, 0)
+    assert messages[1]['time'] == datetime(2018, 1, 2, 0, 0)
+    assert messages[0]['message'] == 'Sample Message'
+    assert messages[1]['message'] == 'Sample Message 2'
+    assert messages[0]['poster'] == 'Davis'
+    assert messages[1]['poster'] == 'Davis'
```

Comment:
Would be clearer to do `assert messages == [{...}, {...}]`, and similarly in other assertions